File: geek-university/algoritmo-e-logica-de-programacao/python/exercicio_24.py
idade = int(input("Informe sua idade: "))

if 5 <= idade <= 7:
    print("Infantil A")
elif 8 <= idade <= 11:
    print("Infantil B")
elif 12 <= idade <= 13:
    print("Juvenil A")
elif 13 <= idade <= 17:
    print("Juvenil B")
elif idade > 18:
    print("Adulto")
else:
    print("Idade abaixo do limite")

File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_24.c
#include <stdio.h>

int main() {
    int idade;

    printf("Informe sua idade: ");
    scanf("%d", &idade);

    if (idade >= 5 && idade <= 7) {
        printf("Infantil A");
    } else if (idade >= 8 && idade <= 11) {
        printf("Infantil B");
    } else if (idade >= 12 && idade <= 13) {
        printf("juvenil A");
    } else if (idade >= 14 && idade <= 17) {
        printf("Juvenil B");
    } else if (idade > 18) {
        printf("Adulto");
    } else {
        printf("Idade abaixo do limite");
    }
}
